pipelines/tasks/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
```

Comment:
Sorry, I wasn't clear -- I wasn't proposing to use `extends` here, but rather to move this compose file up to the top level and rename it something like `docker-compose.tasks.yml`; then we could invoke `docker-compose -f docker-compose.yml -f docker-compose.tasks.yml` to run tasks, which would make it clear where everything is getting called from, and it would also allow us to use `link` as normal.

As far as using both sets of containers, I think it really comes down to how we decide to use Luigi, i.e. whether we use `assistant` or not. If we use `assistant` then I definitely agree that the two compose files should be combined because they will both be services at that point. However, I think that designing our architecture around `assistant` is risky because it's a new, relatively untested feature that seems to run against Luigi's execution model; relying on it will make it more likely that we have a bad experience with Luigi and have to switch later. A lot of Luigi's features that we like come about as a result of its different execution model as compared to Celery / SWF, so if we think that these features are compelling enough to make us want to try Luigi, I think we should try to use it the way it was originally designed. Then if we have a bad experience we'll at least know it was the framework and not because we relied on a feature that wasn't sufficiently exercised.